synapse_data_warehouse/database_roles/V2.39.0__proxy_admin_role.sql

```diff
@@ -0,0 +1,13 @@
+USE DATABASE {{ database_name }}; --noqa: JJ01,PRS,TMP
+
+-- Create proxy admin database role which will own the `*ALL_ADMIN` roles
+CREATE OR REPLACE DATABASE ROLE ALL_ADMIN;
+
+-- Grant ownership of the proxy admin database role to the database admin
+GRANT OWNERSHIP
+    ON DATABASE ROLE ALL_ADMIN
+    TO ROLE {{ database_name }}_ADMIN; --noqa: JJ01,PRS,TMP
+
+-- Grant proxy admin role to the database admin account role
+GRANT DATABASE ROLE ALL_ADMIN
```

Comment:
Is it redundant to grant USAGE of the DB role to <domain>_ADMIN if it's already the owner?

Reply:
No, ownership doesn't give you usage of a role. Just like how SECURITYADMIN is meant to be the owner of all roles, but they don't inherit any of those roles privileges for themself.

Reply:
Noted - thanks!